holoviews/plotting/bokeh/raster.py

```diff
@@ -156,10 +167,404 @@ def get_data(self, element, ranges, style):
                                    for v in aggregate.dimension_values(vdim)]
+
+        # Filter radial heatmap options
+        style = {k: v for k, v in style.items() if not
+                 any(g in k for g in RadialHeatMapPlot._style_groups.values())}
         return (data, {'x': x, 'y': y, 'fill_color': {'field': 'zvalues', 'transform': cmapper},
                        'height': 1, 'width': 1}, style)
 
 
+class RadialHeatMapPlot(CompositeElementPlot, ColorbarPlot):
+
+    clipping_colors = param.Dict(default={'NaN': 'white'}, doc="""
+        Dictionary to specify colors for clipped values.
+        Allows setting color for NaN values and for values above and below
+        the min and max value. The min, max, or NaN color may specify
+        an RGB(A) color as a either (1) a color hex string of the form
+        #FFFFFF or #FFFFFFFF, (2) a length-3 or length-4 tuple specifying
+        values in the range 0-1, or (3) a named HTML color.""")
+
+    start_angle = param.Number(default=np.pi/2, doc="""
+        Define starting angle of the first annulus segment. By default, begins 
+        at 12 o'clock.""")
+
+    padding_inner = param.Number(default=0.1, bounds=(0, 0.5), doc="""
+        Define the radius fraction of inner, empty space.""")
+
+    padding_outer = param.Number(default=0.05, bounds=(0, 1), doc="""
+        Define the radius fraction of outer space including the labels.""")
+
+    xmarks = param.Parameter(default=None, doc="""
+        Add separation lines between segments for better readability. By
+        default, does not show any separation lines. If parameter is of type 
+        integer, draws the given amount of separations lines spread across 
+        radial heatmap. If parameter is of type list containing integers, show 
+        separation lines at given indices. If parameter is of type tuple, draw 
+        separation lines at given segment values. If parameter is of type 
+        function, draw separation lines where function returns True for passed 
+        segment value.""")
+
+    ymarks = param.Parameter(default=None, doc="""
+        Add separation lines between annulars for better readability. By
+        default, does not show any separation lines. If parameter is of type 
+        integer, draws the given amount of separations lines spread across 
+        radial heatmap. If parameter is of type list containing integers, show 
+        separation lines at given indices. If parameter is of type tuple, draw 
+        separation lines at given annular values. If parameter is of type 
+        function, draw separation lines where function returns True for passed 
+        annular value.""")
+
+    max_radius = param.Number(default=0.5, doc="""
+        Define the maximum radius which is used for the x and y range extents.
+        """)
+
+    radial = param.Boolean(default=True, doc="""
+        Whether the HeatMap should be radial""")
```

Comment:
Maybe ``max_radius`` could just be ``radius`` and a none ``None`` value would effectively switch ``radial`` to True? Probably nicer than using ``start_angle`` for this purpose...

Reply:
Also dont like this overloading and doesn't seem much more intuitive than using the angle. Both are options you wouldnt necessarily set and it's not immediately clear what you'd set it too.